csp-mixin/README.md

```diff
@@ -0,0 +1,17 @@
+# CSPlib
+
+A collection of dashboards and their component parts for cloud service provider SaaS services.
```

Comment:
It'd be good to have a sample screenshot here, just to give a quick overview of what's included